draft-ietf-rats-architecture.md

```diff
@@ -1317,3 +1314,160 @@ use beyond the period for which it deems the Attestation Result to remain
 valid.  Thus, if the Attestation Result sends a validity lifetime
 in terms of time(RX)-time(RG), then the Relying Party can check
 `time(OP) - time(ER) < time(RX)-time(RG)`.
+
+# Appendix B: System Examples
+
+(Rough first draft to see if there is interest)
+
+## System Example with OS Vendor as Main Driver
+
+Three entities (companies / organizations) participate in this examples.
+
+The operating system vendor is at the center. It sets the architecture and the level of security achieved. It performs the following:
```

Comment:
It isn't clear which attestation role the operating system vendor is assuming. As an endorser that provisions the attestation private key and the mfg CA that issues the attestation ID certificate, this often isn't an operating system vendor. Since this is an example, the vendor very well could be, but that isn't usually how it works.